contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,133 @@
+| Fonts: Downloading and Rendering  | 🟡 Medium (only if they are part of the initial content)  | 🔴 Low |  🔴 Low  |
+
+## Identifying opportunities on your own site
+PostHog's performance monitoring tool shows you the size and speed of different requests made during page load. You can also filter these by request type, such as API, JS, image, and CSS requests.
+
+Combine this with the cheat sheet above and you'll be able to pinpoint what are the contributing factors to your metric times.
+
+### **Examples**
+**1. API requests**
+![A screenshot of the network tab filtered to show API requests only. One API request is identified as taking too long.](https://i.imgur.com/JPeZeTe.png)
+
+Here we can see an API request that is happening during page load (indicated by the `LOAD` text next to it). Since it is blocking the initial rendering, it is contributing to the high FCP time as well as the Page Loaded time.
+
+**2. JS requests**
+
+![A screenshot of the network tab filtered to show JavaScript requests only](https://i.imgur.com/klPHLwK.png)
+In this example we see fast times for downloading JavaScript. Expanding into one of the requests, we can also see that the request is not blocking the render of the screen. Thus we can conclude that download JS is not contributing significantly to any of our metrics.
+
+**3. Image requests**
+![A screenshot of the network tab filtered to show image requests only. The list shows small images that are taking more than one second each to download.](https://i.imgur.com/4b45d2k.png)
+
+This example shows tiny images that are taking in excess of one second to download. This may be caused by server or network latency.
+
+## Turning opportunities into results
+
+Once you have identified which factors are affected your page load times, here are some actionable next steps to improve them:
+
+### JavaScript
+* Optimize the order of files by ensuring that critical scripts load first.
+* Defer or asynchronously load non-critical JavaScript.
+* Minify files i.e. strip out comments, line breaks and other unnecessary characters. You can use tools such as [Google's Closure Compiler](https://developers.google.com/closure/compiler) or [UglifyJS](https://github.com/mishoo/UglifyJS).
+
+### Images
+* Use image compression tools like [TinyPNG](https://tinypng.com/) or [ImageOptim](https://imageoptim.com/) to reduce file sizes without sacrificing quality.
+* Choose modern image formats like WebP or AVIF, which offer better compression ratios than traditional formats like JPEG and PNG.
+* Implement lazy-loading 
+* Use responsive images with the 'srcset' attribute to serve the right-sized image based on the user's screen size.
+
+### Server and network latency
+* Use a Content Delivery Network (CDN). CDNs cache your static assets across multiple servers around the world, allowing users to access your content from the server closest to them.
+* Implement Load Balancing.
```

Comment:
Additional suggestion would be something like "Minimise your response payloads to return as little data as is needed for the frontend". 

This is the most common think we have discovered via the Network panel - which APIs are responding with Mb+ of data and then reducing that dramatically had the biggest impact

Reply:
@benjackwhite 

> This is the most common think we have discovered via the Network panel - which APIs are responding with Mb+ of data and then reducing that dramatically had the biggest impact

Thanks! I think this could be a great screenshot to include in the post. Would you happen to have one?

Reply:
![Screen Shot 2023-04-11 at 9 11 46 AM](https://user-images.githubusercontent.com/34755028/231224220-da765735-033d-4864-be21-308c9f59302d.png)

Here's an example of zipped vs non-zipped responses from the network panel in the PostHog app. It shows the difference 😄

Reply:
Wow! Such a big difference. I'll include in the blog, thanks!